app/resources/results.html.template

```diff
@@ -82,214 +82,233 @@
-        {# ui=light. #}
-        {# <img align='left'> and <br clear='all'> are for cases where CSS #}
-        {# is disabled e.g. on ui=light. #}
-        {% if result.photo_url %}
-          <div class="resultImageContainer"
-            ><img class='resultImage' src='{{result.photo_url}}'
-                  width='80' height='80' align='left' alt=''
-          /></div>
-        {% else %}
-          <div class="resultImageContainer"
-            ><img class="resultImage" src="{{env.global_url}}/no-photo.gif"
-                  width='80' height='80' align='left' alt=''
-          /></div>
-        {% endif %}
-        <div class='resultData'>
-          {% if env.enable_dup_mode %}
-            <input style='display:none; float:{{env.rtl|yesno:"right,left"}}'
-                type='checkbox' name='dup' value='{{result.record_id}}'
-                onclick='on_dup_click(event, this);' class='duplicate-checkbox'>
+        <div class="button-cancel">
+          <input type="button" onclick="return enable_dup_mode(this);" value="{% trans "Mark records as duplicate" %}">
+        </div>
+      {% endif %}
+      <p>
+      <div class='searchResults'>
+      {% for result in results %}
+        {% ifchanged result.is_address_match %}
+          {% if result.is_address_match %}
+            <hr style="margin-top:30px;">
+            <h1>{% blocktrans with params.query as query %}Records with names and addresses matching "{{query}}"{% endblocktrans %}</h1>
           {% endif %}
-          <a class="result-link" href="{{result.view_url}}">
-            <span class='resultDataTitle'>
-              {{result.full_name_list|join:", "}}
-              {% if config.use_alternate_names and result.alternate_names %}
-                ({{result.alternate_names_list|join:", "}})
-              {% endif %}
-            </span>
-          </a>
-          {% if result.home_street or result.home_neighborhood or result.home_city or result.home_state or result.home_postal_code %}
-          <div>{% trans "Home address" %}:
-              {% if result.home_street %}
-                {{result.home_street}}&nbsp;&nbsp;
-              {% endif %}
-              {% if result.home_neighborhood %}
-                {{result.home_neighborhood}}&nbsp;&nbsp;
-              {% endif %}
-              {% if result.home_city %}
-                {{result.home_city}}&nbsp;&nbsp;
-              {% endif %}
-              {% if result.home_state %}
-                {{result.home_state}}&nbsp;&nbsp;
-              {% endif %}
-              {% if result.home_postal_code %}
-                {{result.home_postal_code}}&nbsp;&nbsp;
-              {% endif %}
-          </div>
+        {% endifchanged %}
+        <div class="resultItem" data-url="{{result.view_url}}" data-status="{{result.latest_status}}"
+            onclick="on_result_item_click(event, this);">
+          {# No whitespace inside .resultImageContainer. #}
```

Comment:
I'm not actually sure what's going on here (I didn't edit this section, just indented it).
Is this stating that we shouldn't put whitespace into the div? Adding whitespace doesn't seem to cause any problems.